configs/openmrs_config/concepts/assessment.csv

```diff
@@ -1,4 +1,4 @@
 Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:km,Short name:en,Short name:km,Description:en,Description:fr,Data class,Data type,Answers,Members,_version:1,_order:1001
-dff491e3-b14c-4b32-ab64-63b96966d72d,,,CVD risk score (%),ពិន្ទុនៃការវាយតម្លៃហានិភ័យជំងឺសរសៃឈាម-បេះដូង,CVD risk score (%),ពិន្ទុនៃការវាយតម្លៃហានិភ័យជំងឺសរសៃឈាម-បេះដូង,,,Question,Numeric,,,,
+dff491e3-b14c-4b32-ab64-63b96966d72d,,,CVD risk score (%),ពិន្ទុនៃការវាយតម្លៃហានិភ័យជំងឺសរសៃឈាម-បេះដូង  (%),CVD risk score (%),ពិន្ទុនៃការវាយតម្លៃហានិភ័យជំងឺសរសៃឈាម-បេះដូង  (%),,,Question,Numeric,,,,
```

Comment:
Why the 2 spaces?

Reply:
Had missed out on that. Had abit of trouble with the commits here is the final one to consider --> https://github.com/openmrs/ozone-distro-cambodia/pull/11/commits/0eda3cf2a148a65b3056ffb27bd53022d05453e3